soh/soh/Enhancements/game-interactor/GameInteractor.h

```diff
@@ -332,6 +332,9 @@ typedef enum {
     VB_PLAY_RAINBOW_BRIDGE_CS,
     // Opt: *EnBox
     VB_PLAY_SLOW_CHEST_CS,
+    // Opt: *ShotSun
```

Comment:
`Opt` is used to indicate the optional arguments passed into the should hook via variadic args. Currently you are not passing in the shot sun actor, so this comment is inaccurate.
Either remove the comment, or pass in the actor (even if it is currently unused in the existing hooks).
(Needs to apply to both enums separately)

Reply:
I thought those were just about where they get used.

Reply:
There are a couple of other misuses from PRs I've done here. Is it OK if I clean those up here too?

Reply:
Yeah if you remember them, thats fine with me.